pkg/cluster/config/types.go

```diff
@@ -42,6 +42,8 @@ type Config struct {
 	ControlPlane *ControlPlane `json:"ControlPlane,omitempty"`
 }
 
+// ControlPlane holds node life cycle hooks to make the actual state of all
```

Comment:
Hmm I'd say something like above instead, ControlPlane holds configuration specific to the "control plane" node(s) (currently the only node)

Reply:
change made